src/test/java/edu/kit/datamanager/repo/util/validators/DOIValidatorTest.java

```diff
@@ -130,7 +125,7 @@ public void serverNotReachable() {
     @Test
     public void invalidPrefixInURL() {
```

Comment:
prefixToLong
prefixToShort
prefixWithInvalidCharacter1
prefixWithInvalidCharacter2
Instead of writing one test for each invalid character there is also the possibility to define an array of invalid characters and test them in a loop with a catch inside.